cmd/kn/main.go

```diff
@@ -62,6 +64,11 @@ func run(args []string) error {
 
 	pluginManager := plugin.NewManager(config.GlobalConfig.PluginsDir(), config.GlobalConfig.LookupPluginsInPath())
 
+	// Merge plugins that implements client-pkg Plugin type to the same manager instance
+	for _, p := range pkgplugin.InternalPlugins {
```

Comment:
I wonder whether we shouldn't put this internally in plugin.NewManager() where it feels more natural to me. It could be a lazy initialization (i.e. do only once regardless how often NewManager() is called). It would also keep `main()` a bit cleaner.

Reply:
https://github.com/knative/client/pull/1816/commits/bfaf3f94b2acdf21505b7bccdfb19362d9eda66b

Reply:
Does it look better now? Agreed on moving the scope to constructor.

Reply:
@rhuss ping if you get to it.